test/sparsedir/sparse.jl

```diff
@@ -499,42 +499,42 @@ for (aa116, ss116) in [(a116, s116), (ad116, sd116)]
     @test ss116[:,:] == copy(ss116)
 
     # range indexing
-    @test full(ss116[i,:]) == aa116[i,:]
-    @test full(ss116[:,j]) == aa116[:,j]
-    @test full(ss116[i,1:2:end]) == aa116[i,1:2:end]
-    @test full(ss116[1:2:end,j]) == aa116[1:2:end,j]
-    @test full(ss116[i,end:-2:1]) == aa116[i,end:-2:1]
-    @test full(ss116[end:-2:1,j]) == aa116[end:-2:1,j]
+    @test convert(Array, ss116[i,:]) == aa116[i,:]
+    @test convert(Array, ss116[:,j]) == aa116[:,j]
+    @test convert(Array, ss116[i,1:2:end]) == aa116[i,1:2:end]
+    @test convert(Array, ss116[1:2:end,j]) == aa116[1:2:end,j]
+    @test convert(Array, ss116[i,end:-2:1]) == aa116[i,end:-2:1]
+    @test convert(Array, ss116[end:-2:1,j]) == aa116[end:-2:1,j]
     # float-range indexing is not supported
 
     # sorted vector indexing
-    @test full(ss116[i,[3:2:end-3;]]) == aa116[i,[3:2:end-3;]]
-    @test full(ss116[[3:2:end-3;],j]) == aa116[[3:2:end-3;],j]
-    @test full(ss116[i,[end-3:-2:1;]]) == aa116[i,[end-3:-2:1;]]
-    @test full(ss116[[end-3:-2:1;],j]) == aa116[[end-3:-2:1;],j]
+    @test convert(Array, ss116[i,[3:2:end-3;]]) == aa116[i,[3:2:end-3;]]
+    @test convert(Array, ss116[[3:2:end-3;],j]) == aa116[[3:2:end-3;],j]
```

Comment:
It's a bad sign when your diffs get long enough that github gives up on coloring words any more. I should probably have looked at this commit-by-commit.